e2e/_screenshot.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const path = require('path');
+const child_process  = require('child_process');
+const mapnik = require('mapnik');
+
+const SHA = process.env.TRAVIS_COMMIT;
+const LFS_BASE_URL = 'https://media.githubusercontent.com/media/angular/visualdiff-dev';
+
+/**
+ * Generates a screenshot from the current state of the Protractor test and compares it to the
+ * previous stored screenshot.  If the screenshots do not match or if no existing screenshot is
+ * found, an error will be thrown.  In both cases, the new screenshot will be stored so that it can
+ * be added to git.
+ */
+class Screenshot {
+  /**
+   * @param {string} id A unique identifier used for the screenshot
+   */
+  constructor(id) {
+    this.id   = id;
+    this.path = path.resolve(__dirname, '..', 'screenshots', id + '.screenshot.png');
+    this.url  = `${ LFS_BASE_URL }/${ SHA }/screenshots/${ encodeURIComponent(id) }.screenshot.png`;
+    browser.takeScreenshot().then(png => this.handleScreenshot(png));
+  }
+
+  /**
+   * Stores the screenshot
+   * @param {image} png The screenshot generated from the current browser state
+   */
+  handleScreenshot(png) {
```

Comment:
Does it make sense to just call this function `storeScreenshot` if that's all it's doing?